src/engine/Algebra.ts

```diff
@@ -68,11 +71,25 @@ module ex {
          return true;
       }
 
+      constructor();
+      /**
+       * @param vectorOptions coordindates of the Vector
+       */
+      constructor(vectorOptions: IVectorOptions);
       /**
        * @param x  X component of the Vector
        * @param y  Y component of the Vector
        */
-      constructor(public x: number, public y: number) { }
+      constructor(x: number, y: number);
```

Comment:
Should these all be marked as obsolete so we can deprecate them in a future version?